test/upgradescheme.js

```diff
@@ -196,8 +196,8 @@ contract('UpgradeScheme', function(accounts) {
       let contractsNames = [];
       let contractsToUpgrade = [];
       for (let i = 0; i < 21; i++) {
```

Comment:
how about make it 60 ? like in the contract limit

Reply:
the loop adds 3 at a time